features/secrets.feature

```diff
@@ -36,7 +36,6 @@ Feature: Secrets
         When a secret "foo" is added globally
         And the "main" job is started
         Then the "foo" secret should be available in the build
-        And the build succeeded
```

Comment:
We said in the description of the PR:
`Remove And the build succeeded since it is already checked in Then the "foo" secret should be available in the build`